thrift/contentatom.thrift

```diff
@@ -0,0 +1,5 @@
+include "atoms/tenfour.thrift"
+
+union ContentAtom {
+  1: tenfour.Content tenfour
```

Comment:
OK. Are we happy to use slightly obscure codenames in our data structures (eg tenfour) rather than more descriptive names (eg quiz), esp if that's going to be expose to third parties in CAPI?

Not a big deal but might as well make the decision consciously.

cc @steppenwells @mchv